### PR TITLE
Remove Jaana Dogan

### PR DESCRIPTION
**Pull request by rakyll**
I decided to remove myself as a nominee given there are many indiviuals here who deeply contributed to the project. I think it'd be wrong to keep my nomination with very tiny direct contributions to the project.

### CODE REVIEW

**Comment by lizthegrey** [comment]
You were still a huge influence on the formation of the project <3

**Review by lizthegrey — APPROVED** [review]

**Comment by rakyll** [comment]
Thanks so much, @lizthegrey! I'm going to make direct contributions again and it'll be better to try this in the future :)

File: elections/2020/governance-committee-candidates.md [hunk 1]
```diff
@@ -49,37 +49,6 @@ Ilan Rabinovitch leads the community and product teams at Datadog. He spends his
 
 ---
 
-### Jaana Dogan
```
Comment (SergeyKanzhelev) [inline elections/2020/governance-committee-candidates.md:52]: would you please also remove you from the list on top?

**Comment by SergeyKanzhelev** [comment]
Thank you for considering yourself for Governance Committee.

**Comment by rakyll** [comment]
PTAL

**Comment by bhs** [comment]
@rakyll since you are suggesting this change yourself I will merge it. Thanks for the fix on the list at the top, too.

>I'm going to make direct contributions again and it'll be better to try this in the future :)

Please do!

**Pull request merged**